Objects/genericaliasobject.c

```diff
@@ -718,16 +727,33 @@ ga_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     if (!_PyArg_NoKeywords("GenericAlias", kwds)) {
         return NULL;
     }
-    if (!_PyArg_CheckPositional("GenericAlias", PyTuple_GET_SIZE(args), 2, 2)) {
+    if (!_PyArg_CheckPositional("GenericAlias", PyTuple_GET_SIZE(args), 2, 3)) {
         return NULL;
     }
+
     PyObject *origin = PyTuple_GET_ITEM(args, 0);
     PyObject *arguments = PyTuple_GET_ITEM(args, 1);
+
+    bool starred;
+    if (PyTuple_GET_SIZE(args) < 3) {
+        starred = false;
+    } else {
+        PyObject *py_starred = PyTuple_GET_ITEM(args, 2);
+        if (!PyBool_Check(py_starred)) {
+            PyErr_SetString(PyExc_TypeError,
+                            "Third argument to constructor of GenericAlias "
+                            "must be a bool (since it's a flag controlling "
+                            "whether the alias is unpacked)");
```

Comment:
```suggestion
                            "must be a bool");
```
We don't need the explanation.